eslint.config.mjs

```diff
@@ -0,0 +1,41 @@
+import eslintPluginJs from '@eslint/js';
+import eslintPluginStylistic from '@stylistic/eslint-plugin';
+import globals from 'globals';
+
+const config = [
+  {
+    files: ['**/*.js', '**/*.mjs'],
+  },
+  {
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        ...globals.node,
+        Log: 'readonly',
+        MM: 'readonly',
+        Module: 'readonly',
+      },
+    },
+    plugins: {
+      ...eslintPluginStylistic.configs['all-flat'].plugins,
+    },
+    rules: {
+      ...eslintPluginJs.configs.recommended.rules,
+      ...eslintPluginStylistic.configs['all-flat'].rules,
+      '@stylistic/array-element-newline': 'off',
+      '@stylistic/brace-style': ['error', '1tbs', {allowSingleLine: true}],
+      '@stylistic/comma-dangle': ['error', 'only-multiline'],
+      '@stylistic/dot-location': ['error', 'property'],
+      '@stylistic/function-call-argument-newline': 'off',
+      '@stylistic/indent': ['error', 2],
+      '@stylistic/padded-blocks': 'off',
+      '@stylistic/quote-props': ['error', 'consistent-as-needed'],
+      '@stylistic/quotes': ['error', 'single'],
+      '@stylistic/multiline-comment-style': 'off',
+      '@stylistic/multiline-ternary': 'off',
+      'object-shorthand': ['error', 'always']
```

Comment:
Is this reconfiguring the default linter to minimize code changes?

Reply:
> Is this reconfiguring the default linter to minimize code changes?

Yes, exactly. I have just reduced this reconfiguration. If we remov the last custom `@stylistic` configuration, that would result in a lot of changes.

Reply:
I think keeping the PR as-is makes perfect sense.

The rest don't bother me either. Feel free to keep it or PR it!